pyproject.toml

```diff
@@ -85,34 +85,34 @@ django-waffle = "^4.1.0"
 nameparser = "^1.1.3"
 eyecite = "*"
 scipy = "^1.14.1"
-scikit-learn = "^1.5.1"
-numpy = "^1.26.3"
-datasketch = "^1.6.0"
+scikit-learn = "^1.5.2"
+numpy = "^2.1.1"
+datasketch = "^1.6.5"
 PyStemmer = "^2.2.0.1"
-factory-boy = "^3.3.0"
+factory-boy = "^3.3.1"
 django-elasticsearch-dsl = "^8.0"
 django-override-storage = "^0.3.2"
 django-environ = "^0.11.2"
 judge-pics = "^2.0.5"
 django-admin-cursor-paginator = "^0.1.5"
 sentry-sdk = {extras = ["celery", "django"], version = "^2.15.0"}
-selenium = "^4.16.0"
-ipython = "^8.27.0"
-time-machine = "^2.14.1"
+selenium = "^4.25.0"
+ipython = "^8.28.0"
+time-machine = "^2.15.0"
 dateparser = "1.2.0"
-types-dateparser = "^1.1.4.20240106"
-uvicorn = {extras = ["standard"], version = "^0.27.1"}
-daphne = "^4.1.0"
+types-dateparser = "^1.2.0.20240420"
+uvicorn = {extras = ["standard"], version = "^0.31.0"}
+daphne = "^4.1.2"
 httpx = {extras = ["http2"], version = "^0.27.2"}
 django-model-utils = "^4.5.1"
 django-permissions-policy = "^4.21.0"
-tiktoken = "^0.6.0"
+tiktoken = "^0.8.0"
```

Comment:
We don't use this directly, it's a dependency of `openai` package, used only on audio transcriptiopns